TKGame/Content/Weapons/Sword.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TKGame.Content.Weapons
```

Comment:
This is a good framework for the weapons. For now having the sword be implemented by the weapons is fine, like what I did for Item. We'll need to make this abstract in the future to have concrete weapons inherit from it.